app/src/main/res/layout-land/add_profile_activity.xml

```diff
@@ -0,0 +1,194 @@
+        android:layout_height="wrap_content"
+        android:layout_marginStart="28dp"
+        android:text="@string/add_profile_required"
+        android:textSize="12sp"
+        app:layout_constraintEnd_toStartOf="@+id/end_guideline"
+        app:layout_constraintStart_toEndOf="@+id/start_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/upload_image_button" />
+
+      <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/pin_container"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:visibility="@{viewModel.createPin ? View.VISIBLE : View.GONE}"
+        app:layout_constraintEnd_toStartOf="@+id/end_guideline"
+        app:layout_constraintStart_toEndOf="@+id/start_guideline"
+        app:layout_constraintTop_toBottomOf="@+id/checkbox_pin">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+          android:id="@+id/allow_download_container"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="64dp"
+          android:layout_marginTop="32dp"
+          android:layout_marginEnd="32dp"
+          app:layout_constraintTop_toBottomOf="@+id/input_confirm_pin">
+
+          <LinearLayout
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="32dp"
+            android:orientation="vertical"
+            app:layout_constraintEnd_toStartOf="@+id/allow_download_switch"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent">
+
+            <TextView
+              android:id="@+id/allow_download_heading"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:text="@string/add_profile_allow_download_heading"
+              android:textColor="@{viewModel.validPin ? @color/black : @color/grey}"
+              android:textStyle="bold" />
+
+            <TextView
+              android:id="@+id/allow_download_sub"
+              android:layout_width="match_parent"
+              android:layout_height="wrap_content"
+              android:text="@string/add_profile_allow_download_sub" />
+          </LinearLayout>
+
+          <Switch
+            android:id="@+id/allow_download_switch"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:clickable="@{viewModel.validPin ? true : false}"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintTop_toTopOf="parent" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+
+        <org.oppia.app.profile.ProfileInputView
+          android:id="@+id/input_confirm_pin"
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="@dimen/margin_32"
```

Comment:
Use direct values instead of dimen values for margins.

Reply:
I didn't write this by myself. I copied it from the portrait mode layout file, so I think I don't need to change them, am I correct?